src/scicookie/cookiecutter.json

```diff
@@ -46,12 +46,15 @@
   "build_system": [
     "poetry"
   ],
+  "linter": [
```

Comment:
keep all tools as `use_my_tool` standard, instead of creating a group.
with the new approach using the CLI and profiles, the options would be defined in a different way
the idea is to keep everything flexible .. so if the user want to have ruff and flake8, why not?

Reply:
in another words, remove `linter` here, move back flake8 as use_flake8 and add `use_ruff`